incident-response-plan.md

```diff
@@ -0,0 +1,153 @@
+# Incident Response Plan
+
+## Purpose  
+This document outlines the process for handling security-impacting incidents affecting one or more projects in the Foundation’s ecosystem. Incidents may include platform changes with security implications, account compromise, or other security events that require coordinated response.
+
+The Foundation’s role is to:  
+1. **Receive and triage THE REPORTS**  
```

Comment:
Suggest we just say "Receive and triage security reports"

Reply:
So far seems like "Receive and triage the Incident Reports" is the final one in 06a3aca22b65e449af709584ce58cddbf4398553